spec/compiler/codegen/arithmetics_spec.cr

```diff
@@ -151,4 +159,78 @@ describe "Code gen: arithmetics primitives" do
       end
     {% end %}
   end
+
+  describe ".to_i conversions" do
+    {% for method, path_type in SupportedIntsConversions %}
+      {% type = path_type.resolve %}
+
+      {% if ![UInt64, Int128, UInt128].includes?(type) %}
```

Comment:
`unless` is supported in macros.

Reply:
I know, but it's not worse. Before there was `||` / `&&` and I parse easier the `if` than the `unless` when they are present.

I think I prefer to keep `unless` when the have no body and stop de execution a block. Like `return unless C` or `break unless C`, because then you know that `C` holds through the rest of the statements. I found it more intuitive, but is a personal choice.